src/classes/modules/java.base/java/lang/Class.java

```diff
@@ -57,14 +52,14 @@ public final class Class<T> implements Serializable, GenericDeclaration, Type, A
 
    // we init this on demand (from MJIEnv) since it's not used too often
   private static Annotation[] emptyAnnotations; // = new Annotation[0];
-  
```

Comment:
We would like to separate non-functional changes from code changes so we can merge this patch with other branches in the future, so please try to not change whitespace.